build/ci/unit-tests.yml

```diff
@@ -38,102 +38,23 @@ pr:
     - README.md
 
 jobs:
-- job: Windows
-  pool:
-    name: NetCorePublic-Pool
-    queue: buildpool.windows.10.amd64.vs2019.pre.open
-  strategy:
-    maxParallel: 4
-    matrix:
-      Debug:
-        _configuration: Debug
-      Release:
-        _configuration: Release
-  timeoutInMinutes: 20
-  steps:
-    - script: $(Build.SourcesDirectory)\build.cmd /build /test /ci /sign /diagnostic /no-deploy /no-integration /no-ibc /configuration $(_configuration)
```

Comment:
This used to not pass `/clearnugetcache`, but will now have that option.

Reply:
See description.